internal/k8s/task_queue.go

```diff
@@ -103,30 +102,19 @@ func (tq *taskQueue) Shutdown() {
 // kind represents the kind of the Kubernetes resources of a task
 type kind int
 
+// resources
 const (
-	// ingress resource
```

Comment:
Nice job, I don't think these comments communicated anything the constant name didn't.